crest/Assets/Crest/Crest/Scripts/LodData/LodDataMgrClipSurface.cs

```diff
@@ -60,6 +60,8 @@ public override void BuildCommandBuffer(OceanRenderer ocean, CommandBuffer buf)
                 return;
             }
 
+            _targetsClear = false;
```

Comment:
Interesting.. targetsClear is anyway written below the loop, so setting it false here only impacts the loop.. I think.. Is the value used within the loop, hence setting it here fixes it? Just trying to get my head around what is going wrong.

Reply:
Apologies. I made a mistake. Only flow was the problem. I have rebased with the correct fix for flow.